src/main/java/de/tum/in/www1/artemis/config/StaticCodeAnalysisConfigurer.java

```diff
@@ -11,6 +13,7 @@
 /**
  * Provides hard-coded programming language specific static code analysis default categories as an unmodifiable Map
  */
+@Profile("core")
```

Comment:
This is a utility class without any spring-related functionality if I'm not mistaken